Harkirat-Singh/Harkirat-Singh.md

```diff
@@ -0,0 +1,72 @@
+
+**Hark** is a Software Engineer I at **SSW**.
+
+He began his journey at SSW as a **FireBootCamp intern**, where he learned and practiced **Angular**, **.NET**, **Clean Architecture**, **Azure DevOps**, and **full-stack development**. After successfully completing the program, he joined SSW full-time, applying the skills and discipline gained during the bootcamp to real-world projects.
+
+## Techy Hark
+* **Flutter** - Primary focus; passionate about crafting beautiful, performant mobile apps.  
+* **Backend-as-a-Service (BaaS) Platforms** - Proficient in **Firebase** and **Supabase**, leveraging them for authentication, storage, and real-time data handling.  
+* **Web Development**  
+  * **Angular** for front-end development  
+  * **.NET (C#)** for backend APIs using **Clean Architecture** principles  
+* **DevOps & CI/CD** - Experienced with **GitHub Actions**, **Azure DevOps**, and continuous integration and deployment workflows.  
+* **Web3 & Blockchain** - Explored **Solidity**, **Ethereum**, and **Polygon** chains, and successfully deployed his own **NFT collection**.  
+* **App Store Experience** - Experienced in publishing apps on both the **Apple App Store** and **Google Play Store**.  
+
+## Beyond Tech
+* **Tales of Hark - YouTube Channel** for sharing different aspects of Hark’s life, including technical tutorials and personal stories.  
+* **Musical Hark** - Plays **tabla** at the Sikh temple.  
+* **Talkative** - Loves to talk and can chat on random topics for hours (a good listener too!).  
+
+## Some Recommendations
+#### 🎬 Movies
+* All Studio Ghibli films  
+* The Dark Knight Trilogy  
+* The Godfather I & II  
+* The Irishman  
+* Forrest Gump  
+* 3 Idiots  
+* Bhaag Milkha Bhaag  
+* Schindler’s List  
+* Gangs of Wasseypur  
+* Border  
+
+#### 📺 Shows
+* Avatar: The Last Airbender (Original version)  
+* Friends  
+* The Family Man  
+* Sacred Games  
+
+#### 🎥 Documentaries
+* The Last Dance  
+
+#### 📚 Books
+* *Man’s Search for Meaning* - Viktor E. Frankl  
+* *Sapiens* - Yuval Noah Harari  
+* *How to Not Die Alone* - Logan Ury  
+* *Three Women* - Lisa Taddeo  
+
+#### 🎵 Music
+* *Merry-Go-Round of Life* - Joe Hisaishi (*Howl’s Moving Castle*)  
+* *The Dark Knight Trilogy* background score - Hans Zimmer  
+* *Last Train to Mars* - Dan Lebowitz  
+
+#### 🌟 Inspiring Videos
+* [**How Great I Am**](https://www.youtube.com/watch?v=V6xLYt265ZM)  
+* [**Marlen Tjøsvoll - “Bang Bang (My Baby Shot Me Down)” | The Voice Norway 2025**](https://www.youtube.com/watch?v=kBjCp5LISuU)  
+* [**Steve Jobs' 2005 Stanford Commencement Address**](https://www.youtube.com/watch?v=UF8uR6Z6KLc)  
+* [**Randy Pausch - Last Lecture: Achieving Your Childhood Dreams**](https://www.youtube.com/watch?v=ji5_MqicxSo)  
+* [**Naval Ravikant on The Joe Rogan Podcast**](https://www.youtube.com/watch?v=3qHkcs3kG44)  
+* [**DHH on the Lex Fridman Podcast**](https://www.youtube.com/watch?v=vagyIcmIGOQ)  
```

Comment:
Probably best to keep this for a personal profile page.

I think Books or Videos that are related to the industry or your job can be OK to be listed as long as you describe how they impacted you and your career.

The SSW People page is sent to (potential) clients and if they take a closer look at the people they'll be working with they want the confidence that they get value.